src/main/java/edu/cornell/kfs/rass/batch/ProposalTranslationDefinition.java

```diff
@@ -11,16 +11,29 @@
 import org.kuali.kfs.module.cg.businessobject.Agency;
 import org.kuali.kfs.module.cg.businessobject.Proposal;
 import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.springframework.beans.factory.InitializingBean;
 
+import edu.cornell.kfs.module.cg.CuCGPropertyConstants;
 import edu.cornell.kfs.rass.RassConstants;
 import edu.cornell.kfs.rass.batch.xml.RassXmlAwardEntry;
 import edu.cornell.kfs.rass.util.RassUtil;
 
-public class ProposalTranslationDefinition extends RassObjectTranslationDefinition<RassXmlAwardEntry, Proposal> {
+public class ProposalTranslationDefinition extends RassObjectTranslationDefinition<RassXmlAwardEntry, Proposal>
+            implements InitializingBean {
 
 	protected BusinessObjectService businessObjectService;
 	protected DateTimeService dateTimeService;
 	protected ParameterService parameterService;
+	protected RassPropertyDefinition grantNumberMapping;
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        grantNumberMapping = getPropertyMappings().stream()
+                .filter(propertyMapping -> StringUtils.equals(
+                        CuCGPropertyConstants.GRANT_NUMBER, propertyMapping.getBoPropertyName()))
+                .findFirst()
+                .orElseThrow(() -> new IllegalStateException("No property mapping was found for Grant Number"));
```

Comment:
I think this error only happens if our XML configuration isn't setup correctly.  If so, this error message might be more helpful if it states where the mapping should be setup.

Reply:
Ok, I'll update this with a better error message.